rasa/shared/importers/importer.py

```diff
@@ -276,6 +275,13 @@ def get_nlu_data(self, language: Optional[Text] = "en") -> TrainingData:
             lambda merged, other: merged.merge(other), nlu_data, TrainingData()
         )
 
+    @rasa.shared.utils.common.cached_method
+    def get_config_file_for_auto_config(self) -> Optional[Text]:
+        """Returns config file path for auto-config only if there is a single one."""
+        if len(self._importers) != 1:
+            return None
```

Comment:
Why don't we just always return the first?

Reply:
We return None because we don't want to auto-fill any of the files when there are multiple? Does that make sense? I think auto-fill is valid only for a single file. Multi file auto-fill seems to me more complex and better to leave alone.

Reply:
Can we log a warning when this happens?